### PR TITLE
Package version update - JAVA / MariaDB

### PR DESCRIPTION
**Pull request by mizzioisf**
Updatedb JAVA JRE and MariaDB package version (see https://github.com/informatici/openhospital/pull/205)
Upped OH version

### CODE REVIEW

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

File: oh.ps1 [hunk 1]
```diff
@@ -98,7 +98,7 @@ $global:ProgressPreference= 'SilentlyContinue'
 ##################### OH general configuration ####################
 
 # -> OH_PATH is the directory where Open Hospital files are located
-# OH_PATH="c:\Users\OH\OpenHospital\oh-1.13"
+# OH_PATH="c:\Users\OH\OpenHospital\oh-1.12"
```
Comment (mwithi) [inline oh.ps1:101]: Why back to 1.12?

File: oh.sh [hunk 2]
```diff
@@ -982,7 +982,7 @@ function start_api_server {
 	echo ""
 	
 	#$JAVA_BIN -Djava.library.path=${NATIVE_LIB_PATH} -classpath "$OH_CLASSPATH" org.isf.utils.sms.SetupGSM "$@"
-	#$JAVA_BIN -client -Xms64m -Xmx1024m -cp "bin/openhospital-api-0.1.0.jar:rsc:static" org.springframework.boot.loader.JarLauncher >> ../$LOG_DIR/$LOG_FILE 2>&1
+	#$JAVA_BIN -client -Xms64m -Xmx1024m -cp "bin/openhospital-api-0.0.2.jar:rsc:static" org.springframework.boot.loader.JarLauncher >> ../$LOG_DIR/$LOG_FILE 2>&1
```
Comment (mwithi) [inline oh.sh:985]: Why back to 0.0.2 ?

Comment (mizzioisf) [reply oh.sh]: removed comment, fixed

File: oh.ps1 (hunk 1 above)
Comment (mizzioisf) [reply oh.ps1]: fixed (it's a comment)

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**